apps/python/descriptions/scdbstrip.rst

```diff
@@ -80,8 +80,40 @@ Examples
 
      scdbstrip -d mysql://sysop:sysop@localhost/seiscomp --days 30
 
+* Remove event and waveform quality parameters newer than 30 days
+
+  .. code-block:: sh
+
+     scdbstrip -d mysql://sysop:sysop@localhost/seiscomp --days 30 -i
+
 * Only remove waveform QC parameters older than 30 days but no others
 
   .. code-block:: sh
 
      scdbstrip -d mysql://sysop:sysop@localhost/seiscomp --days 30 --qc-only
+
+* Remove event and waveform quality parameters older than 2000-01-01 12:00:00
```

Comment:
Would one use "older than" in combination with a timestamp or just "after"? Maybe a native speaker can advise. You might know someone ;)

Reply:
you are right, even without asking a native speaker "older than XXXX" sounds strange. I will rephrase with "after than"